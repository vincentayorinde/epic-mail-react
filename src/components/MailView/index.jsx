import React from 'react';

const MailView = ({id, sender, receiver, title, messageBody}) => (
  <div id={id} className="mailView">
            <p><small> From </small><strong>{sender}</strong> <small> to </small> <strong>{receiver}</strong></p>
            <h1>{title}</h1>
            <p>{messageBody}</p>
            <div className="hr"></div>
            <textarea rows="5">Hi Vincent Thanks for your email </textarea>   
  </div>
)

export default MailView;