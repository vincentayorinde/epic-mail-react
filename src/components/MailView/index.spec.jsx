
import React from 'react';
import { shallow } from 'enzyme';
import MailView from './index';

describe('MailView Component', () => {
  it('Should render without errors', () => {
    const component = shallow(<MailView />);
    expect(component.find('.mailView').length).toBe(1);
    expect(component).toMatchSnapshot();
  });
});
