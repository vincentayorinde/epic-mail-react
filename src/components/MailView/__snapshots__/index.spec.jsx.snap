// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MailView Component Should render without errors 1`] = `
<div
  className="mailView"
>
  <p>
    <strong />
     to 
    <strong />
  </p>
  <h1 />
  <p />
  <div
    className="hr"
  />
  <textarea
    rows="5"
  >
    Hi Vincent Thanks for your email 
  </textarea>
</div>
`;
